.github/template-files/config.yml

```diff
@@ -43,7 +43,9 @@ conda/infrastructure:
   - src: templates/releases/RELEASE.md
     dst: RELEASE.md
     with:
-      placeholder: YY.M
+      canary_channel: https://anaconda.org/conda-canary
+      placeholder: YEAR.MONTH.PATCH
+      placeholder_x: YEAR.MONTH.x
```

Comment:
@ForgottenProgramme what do you think? which of the following is the least confusing:

1. `YY.M.[$patch_number]`
1. `YY.M.${PATCH}`
2. `YY.M.PATCH`
2. `YEAR.MONTH.PATCH`
3. other?

Reply:
If I may, I'd say we should stick with the particular `CalVer` variant we're using, so one of:

1. `YY.M.[$patch_number]`
1. `YY.M.${PATCH}`
2. `YY.M.PATCH`

Reply:
Per the examples on https://calver.org/ we should be using `YY.MM.MICRO`

Reply:
@jezdez good callout, hadn't occurred to me to check calver.org

Reply:
```suggestion
      placeholder: YY.MM.MICRO
      placeholder_x: YY.MM.x
```